zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -143,7 +143,7 @@ public static String getRequiredReplName(String text) {
     int scriptHeadIndex = 0;
     for (int i = 0; i < text.length(); i++) {
       char ch = text.charAt(i);
-      if (ch == ' ' || ch == '\n' || ch == '(') {
```

Comment:
should we change this to regex check for whitespace? I mean, what if someone put a tab there?